lib/common/ns.c

```diff
@@ -359,21 +482,59 @@ static subtree_t *STsetFind(Agnode_t *n0)
   return s0;
 }
  
+// Return a subtree that is the "union" of the roots of s0 and s1.
+// * In fact, the only thing that is used in the return value is the heap index of
+//   the newly merged sub_tree.
+// * However, the pointer to the new sub_tree is the same as the root that it replaces,
+//   so the pointer to the merged sub_tree is in fact in the heap.
+//
+// * Find the root of each subtree-- one of these will become the merged tree
```

Comment:
Is the `--` here intended to be an m-dash? Or some suggestion of post-decrement?